src/app/services/central-server.service.ts

```diff
@@ -2653,16 +2653,16 @@ export class CentralServerService {
       );
   }
 
-  public chargingStationsUnlockConnector(chargeBoxId: string, connectorId: number): Observable<ActionResponse> {
+  public chargingStationsUnlockConnector(chargingStationID: string, connectorId: number): Observable<ActionResponse> {
     this.checkInit();
     const body = {
-      chargeBoxID: chargeBoxId,
+      chargingStationID,
```

Comment:
Is this parameter still required or used somewhere? It is given in the URL now right?